cluster-autoscaler/FAQ.md

```diff
@@ -566,27 +570,41 @@ Provisioning Request (abbr. ProvReq) is a new namespaced Custom Resource that ai
 
 #### Enabling ProvisioningRequest Support
 
-1. **Cluster Autoscaler Version**: Ensure you are using Cluster Autoscaler version 1.30.1 or later.
+1. __Cluster Autoscaler Version__: Ensure you are using Cluster Autoscaler version 1.30.1 or later.
 
-2. **Feature Flag**: Enable ProvisioningRequest support by setting the following flag in your Cluster Autoscaler configuration:
+2. __Feature Flag__: Enable ProvisioningRequest support by setting the following flag in your Cluster Autoscaler configuration:
 --enable-provisioning-reques=true.
 
 #### Supported ProvisioningClasses
 
 Currently, ClusterAutoscaler supports following ProvisioningClasses:
 
-* `check-capacity.autoscaling.x-k8s.io`. 
+* `check-capacity.autoscaling.x-k8s.io`.
 When using this class, Cluster Autoscaler performs following actions:
 
-  * **Capacity Check**: Determines if sufficient capacity exists in the cluster to fulfill the ProvisioningRequest.
+  * __Capacity Check__: Determines if sufficient capacity exists in the cluster to fulfill the ProvisioningRequest.
 
-  * **Reservation from other ProvReqs** (if capacity is available): Reserves this capacity for the ProvisioningRequest for 10 minutes, preventing other ProvReqs from using it.
+  * __Reservation from other ProvReqs__ (if capacity is available): Reserves this capacity for the ProvisioningRequest for 10 minutes, preventing other ProvReqs from using it.
 
-  * **Condition Updates**:
+  * __Condition Updates__:
   Adds a Accepted=True condition when ProvReq is accepted by ClusterAutoscaler and ClusterAutoscaler will check capacity for this ProvReq.
   Adds a Provisioned=True condition to the ProvReq if capacity is available.
   Adds a BookingExpired=True condition when the 10-minute reservation period expires.
 
+* `best-effort-atomic-scale-up.autoscaling.x-k8s.io` (supported from Cluster Autoscalere version 1.30.2 or later).
+When using this class, Cluster Autoscaler performs following actions:
+
+  * __Capacity Check__: Check which pods could be scheduled on existing capacity.
+
+  * __ScaleUp Request__: Evaluates if scaling up a node group could fulfill the remaining requirements of the ProvisioningRequest..
```

Comment:
Could we add more details about the attempted atomiticity here? E.g.:
* That a scale-up will only be attempted if it would accommodate all of the pods from the PR.
* That the scale-up will use the atomic version of `IncreaseSize` if a given cloud provider supports it.
* That the points above don't guarantee atomicity, but try to achieve it (the "best effort" part) and in practice it'll depend on how atomic the `AtomicIncreaseSize` API actually is.

Reply:
Updated

Reply:
Thanks!